media-gfx/povray/povray-3.7.0.8.recipe

```diff
@@ -82,16 +82,17 @@ BUILD()
 	export LDFLAGS="-lbsd -lnetwork"
 	export CXXFLAGS="-std=c++11 -DBOOST_NO_CXX11_CONSTEXPR -D_BSD_SOURCE"
 
-	runConfigure ./configure \
+if [ $effectiveTargetArchitecture = x86_gcc2 ]; then
+	BoostLibLctr=/boot/system/lib/x86
+else
+	BoostLibLctr=`finddir B_SYSTEM_LIB_DIRECTORY`
+fi
+
+runConfigure ./configure \
```

Comment:
This and the if else fi block needs a tab.

Reply:
Looks OK compared to other recipes.